### PR TITLE
feature: add RBAC support

### PR DESCRIPTION
**Pull request by beav**
This commit adds support for calling the RBAC service. Clients are
expected to use the view helper.

### CODE REVIEW

**Review by ehelms — APPROVED** [review]

File: kerlescan/view_helpers.py [hunk 1]
```diff
@@ -46,6 +47,25 @@ def ensure_account_number(request, logger):
             )
 
 
+def ensure_has_role(
+    role, application, app_name, request, logger, request_metric, exception_metric
```
Comment (ehelms) [inline kerlescan/view_helpers.py]: Given the number of arguments that this method takes, there would be an advantage to using keyword parameters. That would make the readability of the caller easier.

Comment (beav) [reply kerlescan/view_helpers.py]: updated

**Comment by beav** [comment]
updated; will update other PRs once this is merged

**Comment by beav** [comment]
oops, didnt see you already approved! i'll update the other PRs monday am

**Pull request merged**